### DISCUSSION TITLE
Cyrillic text recognition

### DISCUSSION THREAD

**Discussion by KaraElan** [PaddlePaddle/PaddleOCR#13309] [Q&A]
Hello

I am trying to recognize text written with Cyrillic alphabet (Russian, to be exact). Yet when I enable "rec""cyrillic" I still get Latin characters in the answer and, as far as I see, "dict_path": "./ppocr/utils/dict/cyrillic_dict.txt" includes Latin. It drastically decreases the recognition accuracy as Latin and Cyrillic alphabets contain many similarly looking characters.

Is it possible to get an answer containing only Cyrillic characters automatically without retraining the model or doing some custom postprocessing?

**Comment by GreatV** [comment]
you may try `ru` model
https://github.com/PaddlePaddle/PaddleOCR/blob/main/ppocr/utils/dict/ru_dict.txt
https://github.com/PaddlePaddle/PaddleOCR/discussions/12734
